admin/user.go

```diff
@@ -149,6 +149,21 @@ func (s *Service) InviteUserToOrganization(ctx context.Context, email, inviterID
 	return nil
 }
 
+func (s *Service) NotifyUserAdditionToOrganization(ctx context.Context, email, orgName, roleName string) error {
+	// Validate email address
+	_, err := mail.ParseAddress(email)
```

Comment:
Should I move this validation to start of `AddOrganizationMember` and `AddProjectMember` so that its not need in downstream methods like this and invitation method.

Reply:
Also it will be good to validate before entering it in the database.

Reply:
moved it to Add methods